FWCore/ParameterSet/python/Config.py

```diff
@@ -3781,5 +3920,304 @@ def testProcessFragment(self):
             p = Process('PROCESS')
             p.extend(f)
             self.assertTrue(hasattr(p,'fltr'))
+        def testProcessAccelerator(self):
+            proc = Process("TEST")
+            p = TestMakePSet()
+            proc.fillProcessDesc(p)
+            self.assertFalse(p.values["options"][1].values["accelerators"][0])
+            self.assertTrue(["cpu"], p.values["options"][1].values["accelerators"][1])
+            self.assertFalse(p.values["@available_accelerators"][0])
+            self.assertTrue(["cpu"], p.values["@available_accelerators"][1])
+
+            proc = Process("TEST")
+            self.assertRaises(TypeError, setattr, proc, "processAcceleratorTest", ProcessAcceleratorTest())
+            proc.ProcessAcceleratorTest = ProcessAcceleratorTest()
+            self.assertEqual(proc.dumpPython(),
+"""import FWCore.ParameterSet.Config as cms
```

Comment:
This is a pretty big dump. Maybe we should consider adding a function that filters out the 'standard' stuff?

Reply:
I agree the length of the dump is annoying, `del proc.MessageLogger` appeared to be simple trick to reduce the length for this test case. (we appear to have only one other test case dumping the entire `Process`)